SpywareFilter/sections/specific.txt

```diff
@@ -11,6 +11,7 @@
 ! ||api.radio.*/info/*/user/playlog - breaks playing radio
 !
 ||linsoo.co.kr/linsoo_statistics_visitor.php
+borfilm.com##ul.right-bar
```

Comment:
![image](https://user-images.githubusercontent.com/8361299/62542894-6e1b4780-b865-11e9-93df-fd0fce2182b9.png)

Reply:
Added domain to old rule with `##.count`.